cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/kafkaassigner/KafkaAssignerDiskUsageDistributionGoal.java

```diff
@@ -160,39 +172,52 @@ private boolean isOptimized(ClusterModel clusterModel, double upperThreshold, do
    * @return true if an action has been taken to improve the disk usage of the broker, false when a broker cannot or
    * does not need to be improved further.
    */
-  private boolean checkAndOptimize(Broker broker,
+  private boolean checkAndOptimize(SortedSet<BrokerAndSortedReplicas> allBrokers,
+                                   BrokerAndSortedReplicas toOptimize,
                                    ClusterModel clusterModel,
                                    double meanDiskUsage,
                                    double lowerThreshold,
                                    double upperThreshold,
                                    Set<String> excludedTopics) {
     LOG.trace("Optimizing broker {}. BrokerDiskUsage = {}, meanDiskUsage = {}",
-              broker, diskUsage(broker), meanDiskUsage);
-    double brokerDiskUsage = diskUsage(broker);
+              toOptimize.broker(), diskUsage(toOptimize.broker()), meanDiskUsage);
```

Comment:
Overall in this class, it would be good to ensure that each double is printed up to the level of desired decimal digits.